tiny_cnn/io/tensorflow/proto_parser.h

```diff
@@ -0,0 +1,256 @@
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#include <string>
+#include <map>
+#include "tensorflow/core/framework/graph.pb.h"
+using namespace std;
```

Comment:
@Wangyida 
Please don't use ```using namespace``` in header files. It will affect user codes. 
http://stackoverflow.com/questions/5849457/using-namespace-in-c-headers